### DISCUSSION TITLE
(kubernetes-cli) support multiple streams

### DISCUSSION THREAD

**Discussion by rgl** [chocolatey-community/chocolatey-packages#1901] [Ideas]
The kubernetes-cli package should track all the [supported kubernetes release streams, normally the last three minors](https://kubernetes.io/releases/):

> The Kubernetes project maintains release branches for the most recent three minor releases (1.24, 1.23, 1.22).

**Comment by AdmiringWorm** [comment]
The `kubernetes-cli` package already supports streams, and makes the decision of which streams to use based on their versions listed in the [Changelog](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/README.md).

~I did notice it was missing the `json` stream configuration, So I'll get that added.~ NVM: I was just blind

**Reply by AdmiringWorm** [reply]
I do look like the updater has stopped working, and since that is a bug there is no issue required to be created.

If you are interested in working on fixing it, we welcome a PR to fix the issue.

**Reply by rgl** [reply]
@AdmiringWorm Please take a look at https://github.com/chocolatey-community/chocolatey-packages/pull/1952
